display/src/main/scala/uk/ac/wellcome/platform/archive/display/files/DisplayStandaloneFile.scala

```diff
@@ -0,0 +1,57 @@
+package uk.ac.wellcome.platform.archive.display.files
+
+import io.circe.generic.extras.JsonKey
+import uk.ac.wellcome.platform.archive.common.storage.models.{StorageManifest, StorageManifestFile}
+
+// We index individual files into their own index in Elasticsearch.
+//
+// For example, somebody might want to query for all the JP2 files created in
+// the "digitised" space in a given month.
+//
+// We don't want to ingest just the DisplayFile model used in the bags API,
+// because that doesn't have any context about the space of external identifier
+// of this particular bag.
+
+case class DisplayAssociatedBag(
+  space: String,
+  externalIdentifier: String,
```

Comment:
What is this?

Reply:
Can you be more specific? Do you mean the case class "DisplayAssociatedBag", or "externalIdentifier", or something else?

Reply:
Apologies. `externalIdentifier`?

Reply:
It's the identifier in something like Sierra, CALM or whatever, supplied by the user when they initiate an upload. e.g. `b12345678`

See https://github.com/wellcomecollection/storage-service#identifiers for context

Reply:
You've explained this to me, thanks for doing it again!